index.html

```diff
@@ -2322,6 +2258,169 @@ <h2>JSON-LD</h2>
 
       </section>
 
+        <section>
+            <h2>JSON Web Token</h2>
+
+            <p>
+                The verifiable credential data model as described in Section <a href="#core-data-model"></a> can be
+                encoded as a JSON Web Token (JWT) [[!RFC7519]] by doing the following:
+            </p>
+
+            <ul>
+                <li>The signature of the JWT is produced by the issuer of the verifiable credential.</li>
+                <li>Because JWS implicitly provides a verifiable proof of the contained verifiable credentials, the "proof" is no longer needed. Please note that
+                    the "jti" claim represents a nonce value.</li>
+                <li>The IANA JWT claims registry should be enforced whenever possible:</li>
+                <ul>
+                    <li>The "expirationDate" becomes the JWT "exp" claim (epoch time).</li>
+                    <li>The "id" of the verifiable credential becomes the JWT "jti" claim.</li>
+                    <li>The "id" contained in the "claim" object becomes the JWT "sub".</li>
+                    <li>If "created" and "creator" are required, these claims could be added to the JWT payload.</li>
+                    <li>The "issuer" becomes the JWT "iss" claim.</li>
+                    <li>The "issuanceDate" becomes the JWT "iat" claim.</li>
+                </ul>
+            </ul>
+
+            <p>The following is a verifiable credential encoded as a JWT.</p>
+
+            <pre class="example nohighlight" title="Expressing a verifiable credential as a JWT (base64url-encoded)">
+eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJAY29udGV4dCI6WyJodHRwczovL3czLm9yZy8yMDE
+4L2NyZWRlbnRpYWxzL3YxIiwiaHR0cHM6Ly9leGFtcGxlLmNvbS9leGFtcGxlcy92MSJdLCJzdWIiOiJ
+kaWQ6ZXhhbXBsZTplYmZlYjFmNzEyZWJjNmYxYzI3NmUxMmVjMjEiLCJqdGkiOiJodHRwOi8vZG12LmV
+4YW1wbGUuZ292L2NyZWRlbnRpYWxzLzM3MzIiLCJpc3MiOiJodHRwczovL2Rtdi5leGFtcGxlLmdvdi9
+pc3N1ZXJzLzE0IiwiaWF0IjoiMTU0MTQ5MzcyNCIsImV4cCI6IjE1NzMwMjk3MjMiLCJ0eXBlIjpbIlZ
+lcmlmaWFibGVDcmVkZW50aWFsIiwiUHJvb2ZPZkFnZUNyZWRlbnRpYWwiXSwiY2xhaW0iOnsiYWdlT3Z
+lciI6MjF9fQ.TQymZk6gB2zDaReE6UTa9KBp61Te_mto9wWRCd1CqcdAjN0SY12-UhOlsxzn-XhZDNtG
+dTwzY_VURIOGbbBDNgnGbTstrwn8-Soh_qig2lsy7fS5rAzBgmdafJLbaKSCElHbyDIl1Vx4Qfc94-WN
+UbiCTShJwtCI7m82lLZ9mJg
+          </pre>
+
+            <p>The example above contains the following JWT header and payload.</p>
+            <pre class="example nohighlight" title="Verifiable credential JWT header">
+{
+  "alg": "RS256",
+  "typ": "JWT"
+}
+          </pre>
+
+            <pre class="example nohighlight" title="Verifiable credential JWT payload">
+{
+  "@context": [
+    "https://w3.org/2018/credentials/v1",
+    "https://example.com/examples/v1"
+  ],
+  "sub": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+  "jti": "http://dmv.example.gov/credentials/3732",
+  "iss": "https://dmv.example.gov/issuers/14",
```

Comment:
How is the issuer key bound to the message?

Reply:
Yes, that should work, I think.

Reply:
In my mind, key discovery is out of scope of this specification. We are talking about a data model and not about protocols. People could implement that either by DID and DID Document resolution based on the `iss` or even based JWK. These would be part of a different spec.